defaults/openstack/config.pan

```diff
@@ -974,107 +1006,54 @@ final variable OPENSTACK_CEPH_LIBVIRT_SECRET ?= {
   default = To be completed
   note = To be completed
 }
-final variable OPENSTACK_SNMPD_COMMUNITY ?= 'openstack';
+variable OPENSTACK_SNMPD_COMMUNITY ?= 'openstack';
 @use{
   type = To be completed
   default = To be completed
   note = To be completed
 }
-final variable OPENSTACK_SNMPD_LOCATION ?= 'undef';
+variable OPENSTACK_SNMPD_LOCATION ?= 'undef';
 @use{
   type = To be completed
   default = To be completed
   note = To be completed
 }
-final variable OPENSTACK_SNMPD_CONTACT ?= 'root <root@localhost>';
+variable OPENSTACK_SNMPD_CONTACT ?= 'root <root@localhost>';
 @use{
   type = To be completed
   default = To be completed
   note = To be completed
 }
-final variable OPENSTACK_SNMPD_IP ?= PRIMARY_IP;
+variable OPENSTACK_SNMPD_IP ?= PRIMARY_IP;
 
 
 #########################
 # HA Specific Variables #
 #########################
 
 @use{
-  type = string
-  note = The secret to be used for HA RabbitMQ Cluster
-}
-final variable OPENSTACK_RABBITMQ_CLUSTER_SECRET ?= if (OPENSTACK_HA) {error('OPENSTACK_RABBITMQ_CLUSTER_SECRET must be set for high availability');} else {null;};
-@use{
-  type = list
-  default = OPENSTACK_RABBITMQ_HOST
-  note = This is a list of hosts to be used for RabbitMQ
-}
-final variable OPENSTACK_RABBITMQ_HOSTS ?= dict(OPENSTACK_RABBITMQ_HOST,OPENSTACK_RABBITMQ_PORT);
-@use{
-  type = list
-  default = OPENSTACK_MEMCACHE_HOST
-  note = This is a list of hosts to be used for Memcached
-}
-final variable OPENSTACK_MEMCACHE_HOSTS ?= dict(OPENSTACK_MEMCACHE_HOST,OPENSTACK_MEMCACHE_PORT);
-@use{
   type = ipaddress
   note = The floating ip address to use for keepalived
 }
-final variable OPENSTACK_FLOATING_IP ?= if (OPENSTACK_HA) {error('OPENSTACK_FLOATING_IP must be set for high availability');} else {null;};
+variable OPENSTACK_FLOATING_IP ?= if (OPENSTACK_HA) {error('OPENSTACK_FLOATING_IP must be set for high availability');} else {null;};
```

Comment:
FLOATING_IP can mislead as it use on OPENSTACK too for a functionnality OPENSTACK_VIRTUAL_IP or OPENSTACK_KEEPALIVE_IP could be better.